packages/core/test/invoke.test.ts

```diff
@@ -1601,12 +1539,104 @@ describe('invoke', () => {
     });
 
     it('should treat a callback source as an event stream', (done) => {
```

Comment:
Similarly this can be simplified to:
```ts
    it('should treat a callback source as an event stream', (done) => {
      const intervalMachine = createMachine<{
        count: number;
      }>({
        id: 'interval',
        initial: 'counting',
        context: {
          count: 0
        },
        states: {
          counting: {
            invoke: {
              id: 'intervalService',
              src: () =>
                fromCallback((cb) => {
                  const ivl = setInterval(() => {
                    cb({ type: 'INC' });
                  }, 10);

                  return () => clearInterval(ivl);
                })
            },
            always: {
              target: 'finished',
              guard: (ctx) => ctx.count === 3
            },
            on: {
              INC: { actions: assign({ count: (ctx) => ctx.count + 1 }) }
            }
          },
          finished: {
            type: 'final'
          }
        }
      });
      interpret(intervalMachine)
        .onDone(() => done())
        .start();
    });
```